src/DIRAC/WorkloadManagementSystem/Client/Matcher.py

```diff
@@ -179,7 +179,8 @@ def _processResourceDescription(self, resourceDescription):
             if name in resourceDescription:
                 resourceDict[name] = resourceDescription[name]
 
-        if resourceDescription.get("Tag"):
+        # resourceDescription["Tag"] can be ''
+        if "Tag" in resourceDescription:
```

Comment:
That is also a problem for 7.3, isn't it?

Reply:
I don't know. I had not seen it before, but I wasn't looking either :D

Reply:
It could be ported to 7.3, I think. Would you, @andresailer ?

Reply:
Done: #6713